crypto/fipsmodule/cipher/internal.h

```diff
@@ -73,12 +74,46 @@ extern "C" {
 // EVP_CIPH_MODE_MASK contains the bits of |flags| that represent the mode.
 #define EVP_CIPH_MODE_MASK 0x3f
 
+// Set of EVP_AEAD->evp_aead_id identifiers, zero is reserved as the "unknown"
+// value since it is the default for a structure.
+#define AEAD_UNKNOWN_ID 0
```

Comment:
Could check that no unique known `AEAD_EVP` object is given the same ID nor the unknown ID.